pkg/client/clientset.go

```diff
@@ -189,7 +194,8 @@ type ClientSet struct {
 	DocumentClient DocumentClient
 
 	// OpenPipelineClient is a client capable of manipulating openPipeline configs
-	OpenPipelineClient OpenPipelineClient
+	OpenPipelineClient       OpenPipelineClient
+	GrailFilterSegmentClient GrailFilterSegmentClient
```

Comment:
Let's keep this PR only about the requested changes. With this it will be easier to see later what needs to be done to add a new API.

I'll prepare a separate small PR just with this. Is this Ok with you?

Reply:
https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1648